src/include/platform/internal/GenericConfigurationManagerImpl.cpp

```diff
@@ -23,8 +23,8 @@
  *          GenericConfigurationManagerImpl<> template.
  */
 
-#ifndef GENERIC_CONFIGURATION_MANAGER_IMPL_IPP
-#define GENERIC_CONFIGURATION_MANAGER_IMPL_IPP
+#ifndef GENERIC_CONFIGURATION_MANAGER_IMPL_CPP
```

Comment:
These seem to behave more like h than CPP files due to the header guards.

Should they be that instead of CPP files? `#pragma once` may also be appropriate then.

Reply:
It's not safe to include these in more than TU so they are not like header files.

I believe they should be standalone TUs but as the description says, "that's not done here."

The guards serve no purpose since it's not safe to include these from a header.